homeassistant/components/mqtt/strings.json

```diff
@@ -31,6 +31,13 @@
           "transport": "MQTT transport",
           "ws_headers": "WebSocket headers in JSON format",
           "ws_path": "WebSocket path"
+        },
```

Comment:
Instead of splitting config and option flow we could also define a common configuration.